package.json

```diff
@@ -7,7 +7,7 @@
     "husky": "0.14.3",
     "istanbul": "1.1.0-alpha.1",
     "lerna": "2.4.0",
-    "prettier": "1.14.2",
+    "prettier": "1.6.0",
```

Comment:
@lcampos - How come we had to downgrade prettier?

I thought you wanted the newer version to be able to use it with yaml?

Reply:
NVM. I saw your explanation on Slack:

`@nchen it looks like the newest version of prettier (1.14.2) does not wrap lines the same way version 1.6.0 used to. I'll revert to the previous version (1.6.0) since we want to keep the current format of our files`